weaver/processes/wps_package.py

```diff
@@ -164,8 +164,13 @@ def retrieve_package_job_log(execution, job):
         # if the process is a weaver package this status xml should be available in the process output dir
         log_path = get_status_location_log_path(execution.statusLocation, out_dir=out_dir)
         with open(log_path, "r") as log_file:
-            for line in log_file:
-                job.save_log(message=line.rstrip("\n"))
+            log_lines = log_file.readlines()
+        if not log_lines:
+            return
```

Comment:
Simply return or throw exception?

Reply:
Simply return. If the process was not verbose, we won't break its execution for it. Logs will just be minimal.